wiki/Glossary/en.md

```diff
@@ -536,31 +539,30 @@ The warning icon that accompanies nuked beatmaps is the symbol for radioactive m
 
 ### Offset
 
-The offset is a period of time between the beginning of a beatmap's song file and the instant the first beat of the rhythm is heard, from which the position of the first beat is calculated. This is measured in milliseconds (ms).
+A period of time between the beginning of a beatmap's song file and the instant the first beat of the rhythm is heard, from which the position of the first beat is calculated. This is measured in milliseconds (ms).
 
-The offset is the first half of beatmap timing, where the other half is the [BPM](/wiki/bpm).
+The offset is the first half of beatmap timing, where the other half is the [BPM](#bpm).
 
 ### Online Song Offset
 
-The online song offset is an offset that is **already** applied to a beatmap after it was ranked.
+An offset that is **already** applied to a beatmap after it was ranked.
 
 Adjusting your [local song offset](/wiki/local-song-offset) will only add more to this value.
```

Comment:
strange, this link was written to use a redirect but had hyphens like an anchor link
```suggestion
Adjusting your [local song offset](#local-song-offset) will only add more to this value.
```